src/Labeltitle4/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {PropTypes} from 'react';
+import classNames from 'classnames';
+import styles from './styles.css';
+import Typography from '../Typography';
+import Icon from '../Icon';
+
+function labelTitle4(props) {
```

Comment:
the convention is to have the component name start with a capital letter. Your last component was also like this.